### PR TITLE
Feat: Download ERC-721 content

### LINKED ISSUE #1583: ️🛠 - Support downloading content of ERC721 NFTs
### Task description

We should use the downloadUrl/composedUrl to actually download the associated content with the NFT.

### Requirements

_TBD_

### PR DESCRIPTION
**Pull request by Tuditi**
## Summary

Download ERC-721 content so it can be shown as well.
closes: #1583 

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/src/lib/core/nfts/utils/buildNftFromPersistedErc721Nft.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ export function buildNftFromPersistedErc721Nft(nft: IPersistedErc721Nft): INft {
         metadata: {
             contract: nft.contractMetadata,
             ...(tokenMetadata && { token: tokenMetadata }),
+            type: 'image/png',
```
Comment (maxwellmattryan) [inline packages/shared/src/lib/core/nfts/utils/buildNftFromPersistedErc721Nft.ts]: Should use a `MimeType` enum for this?

**Review by maxwellmattryan — CHANGES_REQUESTED** [review]
One small change :pray:

**Review by maxwellmattryan — DISMISSED** [review]
Approving - functionality is working well :+1: nice job

File: packages/shared/src/lib/core/nfts/utils/buildNftFromPersistedErc721Nft.ts (hunk 1 above)
Comment (Tuditi) [reply packages/shared/src/lib/core/nfts/utils/buildNftFromPersistedErc721Nft.ts]: Yup, I cleaned this part up a bit.

**Review by maxwellmattryan — DISMISSED** [review]
LGTM :cherry_blossom:

**Review by maxwellmattryan — APPROVED** [review]

**Pull request merged**